CMakeLists.txt

```diff
@@ -50,6 +50,14 @@ set (cupcfd_VERSION_MINOR 0)
 # ======================================
 # ======== Set Compile Flags ===========
 # ======================================
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+# Use the default CXX compiler if the user hasn't set a COMPILER option
+if (NOT DEFINED${COMPILER} OR ${COMPILER} STREQUAL "")
```

Comment:
Is lack of a whitespace after "DEFINED" potential for problems?

Reply:
Good catch! Fixed.